packages/office-ui-fabric-react/src/components/GroupedList/GroupHeader.base.tsx

```diff
@@ -206,3 +222,7 @@ export class GroupHeaderBase extends React.Component<IGroupHeaderProps, IGroupHe
     );
   };
 }
+
+const FastCheck = React.memo((props: { theme?: ITheme; checked?: boolean; className?: string }) => {
```

Comment:
Why is this called a "FastCheck" - specifically what makes it fast?

Reply:
I copied the interface from <DetailsRowCheck> to be consistent. This applies the "useFastIcons" to the <Icon> component. 
See here: https://docs.microsoft.com/en-us/javascript/api/office-ui-fabric-react/icheckprops?view=office-ui-fabric-react-latest